src/app/admin/semesters/page.tsx

```diff
@@ -0,0 +1,17 @@
+import EditSessionsHeader from "@/components/admin/EditSessionsHeader";
+import SemestersList from "@/components/admin/SemestersList";
+import { Toaster } from "@/components/ui/toaster";
+
+const SemestersPage = () => {
+  return (
+    <div className="relative flex px-4 min-h-dvh max-w-dvw flex-col bg-background dark overflow-x-hidden">
+      <EditSessionsHeader />
```

Comment:
Oops, I think there's been a bit of desync and lack of communication between the different branches, we actually have a `<NavigationBar>` component for this component that was created in another PR.

Reply:
oops

Reply:
So can we delete `EditSessionsHeader` and change this to use the `NavigationBar` component instead?